src/main/generic/consensus/account/tree/AccountsTreeStore.js

```diff
@@ -13,9 +13,17 @@ class AccountsTreeStore {
 }
 Class.register(AccountsTreeStore);
 
+
+let _instance = null;
```

Comment:
We should solve this differently by leveraging a variable like `PersistentAccountsTreeStore._instance`.
Check other singletons in our code (e.g., `WebRtcCertificate`).